core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -7086,6 +7086,7 @@ class KafkaApisTest extends Logging {
   def testConsumerGroupDescribe(): Unit = {
```

Comment:
Should we parameterize the tests to test both variants?

Reply:
good idea

Reply:
@dajac Do you think it is necessary to have two tests, one with includeAuthorizedOperations and one without it? 
I could at it to the IT, without having one IT with includeAuthorizedOperations=false.
Its covered in the unit tests anyways and the option does not change the behavior of the API

Reply:
Yeah, adding it to the existing integration test is fine for me.